src/com/xilinx/rapidwright/edif/EDIFNet.java

```diff
@@ -148,15 +162,36 @@ public EDIFPortInst createPortInst(String portInstName, EDIFCell cell) {
      * @return The newly created port instance or null if none could be created on
      *         the cell or cell instance.
      */
-    private EDIFPortInst createPortInstFromPortInstName(String portInstName, EDIFCell cell, EDIFCellInst inst) {
+    public EDIFPortInst createPortInstFromPortInstName(String portInstName, EDIFCell cell, EDIFCellInst inst) {
+        return createPortInstFromPortInstName(portInstName, cell, inst, false);
+    }
+
+    /**
+     * Creates a port instance from a name. Navigates port naming issues when bussed
+     * names can collide with single bit port names.
+     * 
+     * @param portInstName Proposed name of the new port instance
+     * @param cell         The cell from which to draw the port
+     * @param inst         If this is not null, the port instance is added to the
+     *                     external facing port connection. If this is null, it will
+     *                     add it to the inward facing port connection.
+     * @param parserCreate A flag to indicate if this PortInst is being created
```

Comment:
Maybe a better name for `parserCreate` is `deferSort` as this optimization may be helpful for batching non-parser activities too?